quarto/faq.qmd

```diff
@@ -13,11 +13,36 @@ format:
 #### 1. \ How do I install the falcon package?
 
 For instructions on installing the `falcon` package please see the [Environment Setup and Package Installation](https://pharmaverse.github.io/falcon/quarto/getting_started.html#environment-setup-and-package-installation) section of the Getting Started page.
+<br/><br/>
+
+
+#### 2. \ How can I join this initiative?
+
+For information on how to join the falcon initiative please see the [Joining as a Collaborator](https://pharmaverse.github.io/falcon/quarto/about.html#joining-as-a-collaborator) section of the About page.
+<br/><br/>
+
+
+#### 3. \ I copied the example "Table Setup" code for one of the falcon tables. How can I customize the table?
+
+For each table template page, next to the "Table Setup" tab is the "Function Details" tab. This tab contains information about each of the customizable parameters available for the table, and what the default settings for each are. If you require a customization option that is not available for a given table, please file a Feature Request issue on GitHub [here](https://github.com/pharmaverse/falcon/issues/new/choose).
+<br/><br/>
+
+
+#### 4. \ I need a table that isn't available on the falcon website. Can I request a table be added?
+
+The falcon team is actively maintaining and adding new table functions to the package. If you would like an FDA safety table that is not listed on the falcon website, please file a New Table Template issue on GitHub [here](https://github.com/pharmaverse/falcon/issues/new/choose) with the table number and the team will work on adding it to the package.
+<br/><br/>
+
+
+#### 5. \ I don't know the number of the FDA safety table I an interested in. Is there a list of tables by title?
```

Comment:
Hi Emily, thanks for adding those helpful questions. Only one comment: There's a typo in line 37: ... I a**n** interested ...